.github/workflows/check-licenses.yml

```diff
@@ -27,18 +27,26 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Clone License Check Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           repository: eclipse-velocitas/license-check
-          ref: v1
+          ref: topic/prepare_release
           path: .github/actions/license-check
 
       - name: Run License Checker
+        id: checker
         uses: ./.github/actions/license-check
         with:
           config-file-path: ./.licensechecker.yml
           fail-on-violation: false
-          generate-notice-file: false
+          notice-file-name: "NOTICE-3RD-PARTY-CONTENT"
+
+      - name: Upload notice file if dirty
```

Comment:
Where is described what the developer needs to do if the file is dirty ?

Reply:
"dirty" is common git terminology. This step is just here to upload the notice file to the build artefacts if it was changed due to license scan.

But around this I'm currently "experimenting" to get a good user experience and would like to get your input about.